mysql-test/suite/sys_vars/t/sysvars_innodb.test

```diff
@@ -5,13 +5,13 @@
 
 --vertical_results
 --replace_regex /^\/\S+/PATH/ /\.\//PATH/
+--replace_result 'resides in persistent memory or ' ''
 select VARIABLE_NAME, SESSION_VALUE, DEFAULT_VALUE, VARIABLE_SCOPE, VARIABLE_TYPE, VARIABLE_COMMENT, NUMERIC_MIN_VALUE, NUMERIC_MAX_VALUE, NUMERIC_BLOCK_SIZE, ENUM_VALUE_LIST, READ_ONLY, COMMAND_LINE_ARGUMENT from information_schema.system_variables
   where variable_name like 'innodb%' and
   variable_name not in (
     'innodb_numa_interleave',           # only available WITH_NUMA
```

Comment:
..and this seems to be a collection of all variables that appear/disappear based on platform capabilities, even though the source code and dependency versions would be the same.

Reply:
Typically places that have documentation to users show one version per software version. For example https://manpages.debian.org/unstable/mariadb-server-core/mariadbd.8.en.html has 3 versions for the 3 versions of MariaDB in Debian currently. Now it seems that the text will be different depending on architecture, so documentation should maybe have its own version per architecture.. obviously that would not make sense in any online collection of documentation, but for binaries outputting different help texts on different platforms it is less clear that may be the case